src/fieldExtensions.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * The flattenState() field extension allows the user to access fields normally like
+ * `state.field` instead of `state.present.field`.
+ *
+ * Warning: if your state has fields that conflict with redux-undo's like .past or .index
+ * they will be overridden. You must access them as `state.present.past` or `state.present.index`
+ */
+
+export const flattenState = () => {
+  return (undoableConfig) => {
+    console.log(undoableConfig)
```

Comment:
is this debug output? shouldn't we remove this?

Reply:
Whoops! I will remove this.